thoth/storages/graph/models_base.py

```diff
@@ -73,7 +73,7 @@ def to_dict(self, without_id: bool = True) -> dict:
             if without_id and column.name == "id":
                 continue
 
-            result[column.name] = str(getattr(self, column.name))
+            result[column.name] = str(getattr(self, column.name)) if getattr(self, column.name) else None
```

Comment:
Nice catch! :+1:

Reply:
```
```
{
  "author": "Google Inc.",
  "author_email": "packages@tensorflow.org",
  "description": null,
  "description_content_type": null,
  "download_url": "https://github.com/tensorflow/tensorflow/tags",
  "home_page": "https://www.tensorflow.org/",
  "keywords": "tensorflow tensor machine learning",
  "license": "Apache 2.0",
  "maintainer": null,
  "maintainer_email": null,
  "metadata_version": "2.1",
  "name": "tensorflow",
  "requires_python": null,
  "summary": "TensorFlow is an open source machine learning framework for everyone.",
  "version": "2.0.0"
}
```